tests/unit/lib/test_signal_schema.py

```diff
@@ -356,6 +357,24 @@ def test_slice_nested():
     assert list(sliced.values.items()) == [("feature.aa", int)]
 
 
+def test_mutate_rename():
+    schema = SignalSchema({"name": str})
+    schema = schema.mutate({"new_name": Column("name")})
```

Comment:
why not a standard way `mutate(new_name=Column("name"))` ?

Reply:
`SignalSchema.mutate()` accepts normal dict, not kwargs